student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] You should add `age`, `name`, and `parent_permission` as parameters in `initialize then pass them in `super` 🙃 . Please kindly fix that 😉